### PR TITLE
[DT-041687] Decisions Service fails to restart when 'Financial' module is installed

### PR DESCRIPTION
**Pull request by noah-decisions**
The library ExcelFinancialFunctionsCore is based on ExcelFinancialFunctions (with both of them creating the same DLLs), which was added to the main framework [DT-041078] Implement additional formulas. This caused a conflict between the DLLs when installing the Financial module. I have set the module to use the same library as the framework so that it will be compatible with the DLL there, and removed the DLL and the FSharp.Core DLL (used by the Excel functions DLL) from the modules reference DLLs.

I have confirmed this new version of the module is compatible with version 9.2

### CODE REVIEW

File: Decisions.Financial/Decisions.Financial.csproj [hunk 1]
```diff
@@ -19,9 +19,9 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="DecisionsSDK" Version="8.8.0" />
```
Comment (jason-decisions) [inline Decisions.Financial/Decisions.Financial.csproj]: Please update this to 9.1.0

File: Module.Build.json [hunk 2]
```diff
@@ -4,11 +4,7 @@
       "CoreServicesDlls":[
 		"./Decisions.Financial/obj/Decisions.Financial.dll"
       ],
-      "ReferenceDlls": 
-        [
-          "./Decisions.Financial/obj/ExcelFinancialFunctions.dll",
-          "./Decisions.Financial/obj/FSharp.Core.dll"
```
Comment (jason-decisions) [inline Module.Build.json:10]: Is FSharp.Core.dll in Decisions/bin already?

Comment (noah-decisions) [reply Module.Build.json:10]: After I removed ExcelFinancialFunctions from the Module.Build.json file, I got the same conflicting dll error for FSharp.Core. It's not explicitly added in nuget, but brought along by ExcelFinancialFunctions

File: Decisions.Financial/Decisions.Financial.csproj (hunk 1 above)
Comment (noah-decisions) [reply Decisions.Financial/Decisions.Financial.csproj]: Updated. Also needed to update the project from targeting .Net7 to .Net8 to get the Decisions SDK to update

**Review by jason-decisions — APPROVED** [review]

**Pull request merged**